app/validators/fee/concerns/case_numbers_validator.rb

```diff
@@ -46,12 +46,15 @@ def validate_each_case_number
         return if case_numbers.blank?
 
         case_numbers.split(',').each do |case_number|
-          unless case_number.strip.match?(CASE_NUMBER_PATTERN)
-            add_error(:case_numbers, 'invalid')
-            break
-          end
+          case_number = case_number.strip
+          validate_case_number(case_number)
         end
       end
+
+      def validate_case_number(case_number)
+        add_error(:case_numbers, 'invalid') unless case_number.match?(CASE_NUMBER_PATTERN)
+        add_error(:case_numbers, 'eqls_claim_case_number') if case_number.eql?(claim.case_number)
```

Comment:
```suggestion
        add_error(:case_numbers, 'eqls_claim_case_number') if case_number.casecmp?(claim.case_number)
```

@kmahern I looked in to this issue with Alice and found this (`casecmp?`), it's a case _in_-sensitive comparison operator

Reply:
Thanks @colinbruce will make the change.